src/components/Dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,52 @@
+import DialogTitle from "@mui/material/DialogTitle";
+import Dialog, { DialogProps } from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import { ReactNode } from "react";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import CloseIcon from "@mui/icons-material/Close";
+import Stack from "@mui/material/Stack";
+
+type AlertDialogProps = DialogProps & {
+  title: string
+  content: ReactNode
+  handleClose: () => void
+  cancelText: string
+  confirmText: string
+  onConfirm: () => void
+  icon: JSX.Element
+}
+
+export const AlertDialog = ({ title, content, handleClose, cancelText, confirmText, onConfirm, icon, ...props }: AlertDialogProps) => (
+  <Dialog onClose={handleClose} {...props}>
+    <DialogTitle>
```

Comment:
```suggestion
    <DialogTitle component="div">
```
Otherwise, we get an error we are nesting h6 into an h2 tag.